lib/diff.ml

```diff
@@ -4,17 +4,26 @@ type ('item, 'diff) t = Added of 'item | Removed of 'item | Modified of 'diff
 
 type 'a atomic_modification = { reference : 'a; current : 'a }
 (** The simplest diff representation for the modification of a value of type 'a.
-     [reference] is the value before and [current] is the value after the change occured.
-     Use this type when there is no better representation available. *)
+    [reference] is the value before and [current] is the value after the change
+    occured. Use this type when there is no better representation available. *)
 
 type value = {
   vname : string;
   vdiff : (value_description, value_description atomic_modification) t;
 }
 
+type 'changed type_modification =
+  | Compound of 'changed list
+  | Atomic of type_declaration atomic_modification
+
+and label_ = {
```

Comment:
I don't think label is a keyword so you should be able to go for `label` if you'd like.

That being said, label is used for other things as well in the compiler libs so we could disambiguate this by making it `record_field`. What do you think?

Reply:
`record_field` seems a better name.